From bc97764d64df5a6de4967422d5357811f0ec87ed Mon Sep 17 00:00:00 2001
From: Guillaume Tucker <guillaume.tucker@arm.com>
Date: Wed, 5 Nov 2014 16:08:13 +0000
Subject: [PATCH 6/7] ARM: odroidxu3_defconfig: enable Mali Gator support

Enable MALI_GATOR_SUPPORT and PROFILING which are
needed to use Gator with the Mali GPU.
---
 arch/arm/configs/odroidxu3_defconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/configs/odroidxu3_defconfig b/arch/arm/configs/odroidxu3_defconfig
index dd43284..9bdb263 100644
--- a/arch/arm/configs/odroidxu3_defconfig
+++ b/arch/arm/configs/odroidxu3_defconfig
@@ -2650,7 +2650,7 @@ CONFIG_DRM_UDL=m
 # ARM GPU Configuration
 #
 CONFIG_MALI_MIDGARD=m
-# CONFIG_MALI_GATOR_SUPPORT is not set
+CONFIG_MALI_GATOR_SUPPORT=y
 CONFIG_MALI_MIDGARD_DVFS=y
 CONFIG_MALI_MIDGARD_RT_PM=y
 # CONFIG_MALI_MIDGARD_ENABLE_TRACE is not set
-- 
1.7.9.5

